src/steps/measureSteps.ts

```diff
@@ -17,9 +17,9 @@ export const measureSteps = (mark: string) => {
       const possibleSteps = finalSteps[startMark];
       possibleSteps.forEach(removeActiveStep);
       Promise.all(
```

Comment:
Add comments on what's going on, so in the future we better know the mechanis of this part.